mmset.raw.html

```diff
@@ -4304,7 +4298,9 @@
 <P>
 For the specific case of the set theory database set.mm, Mario Carneiro added
 an enhancement to the <A HREF="../index.html#mmj2">mmj2</A> program that will
-verify the soundness of all but four definitions.  This test can be turned on
+verify the soundness of all but four definitions.  There is a specification
+for this definition checker in the "additional rules for definitions" section
+of the ~ conventions page. The mmj2 definition checker can be turned on
```

Comment:
That page uses the convention "double space after period ending a sentence".

Reply:
Ah, good catch. Fixed.